lib/Completion.groovy

```diff
@@ -111,8 +115,17 @@ class Completion {
         output_tf.withWriter { w -> w << email_txt }
     }
 
-    static void summary(workflow, params, log) {
+    static void summary(workflow, params, log, busco_failed_bins) {
         Map colors = Headers.log_colours(params.monochrome_logs)
+
+        if (busco_failed_bins.size() > 0) {
+            def failed_bins = ''
+            for (bin in busco_failed_bins) {
+                failed_bins += "    ${bin}\n"
+            }
+            log.info "-${colors.purple}[$workflow.manifest.name]${colors.red} For ${busco_failed_bins.size()} bin(s) the BUSCO analysis failed because no genes where found or placements failed:\n${failed_bins}See ${params.outdir}/GenomeBinning/QC/BUSCO/[bin]_busco.err for further information.${colors.reset}-"
```

Comment:
Hm so this is also when `placements failed`, is that the same problem? I am uncertain.

Reply:
yes, in both cases bins would be listed here. 

The problem with the placement error is it says: 
`ERROR:  Placements failed. Try to rerun increasing the memory or select a lineage manually.`

In my case it was definitely not a memory problem. However, I am afraid that this could be caused by memory issues and then we would block retrying (such error messages do not really help...). That is why I pointed to the `${bin}_busco.err` file, so the user could at least discover this. It would of course not be nice, but I also don't know how else to handle this.